frontend/src/app/shared/header/header.component.html

```diff
@@ -1,5 +1,5 @@
 <header>
-  <div>
+  <div class="home-logo" [routerLink]="['home']">
     <img src="assets/images/logo.PNG" alt="logo">
     <trd-subtiredd-select *ngIf="isUserLoggedIn$ | async">
     </trd-subtiredd-select>
```

Comment:
Only the img tag is the logo, not the entire div. Move the router link to img or use navigate in a method on img click. Now the router activates when you click anywhere in the div (subtiredd select included)